docs/content/en/docs/pipeline-stages/cleanup.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Cleanup"
+linkTitle: "Cleanup"
+weight: 60
+featureId: cleanup
+---
+
+Skaffold works with [image builders]({{<relref "/docs/pipeline-stages/builders">}}) and [deployers]({{<relref "/docs/pipeline-stages/deployers">}}) that both have side effects on the environment, namely: 
+
+- resources are created in one or more namespaces in a Kubernetes cluster 
+- images are created on the local Docker daemon and registries
```

Comment:
if we specify remote registries here, we need to add a note that *pushed images wont be deleted* and its up to users to prune them.